### ISSUE TITLE
[mysql error 1064] user_id empty

### ISSUE THREAD

**Issue by Kipjr** [Piwigo/Piwigo#605]
I get the following error message while logging in (for the first time) with an AD-user. LDAP works (tested) but creating a user by using the login field doesn't. Logging in with an existing user does not provide an error. 

```
Warning: array_merge(): Argument #2 is not an array in /var/www/piwigo/include/functions_user.inc.php on line 392

Warning: Invalid argument supplied for foreach() in /var/www/piwigo/include/functions_user.inc.php on line 394

Notice: Undefined index: id in /var/www/piwigo/include/functions_user.inc.php on line 420

Notice: Undefined index: status in /var/www/piwigo/include/functions_user.inc.php on line 420


Warning:  [mysql error 1064] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 3

SELECT cat_id
  FROM piwigo_user_access
  WHERE user_id = 
; in /var/www/piwigo/include/dblayer/functions_mysqli.inc.php on line 845


Fatal error: Uncaught Error: Call to a member function fetch_assoc() on boolean in /var/www/piwigo/include/dblayer/functions_mysqli.inc.php:906 Stack trace: #0 /var/www/piwigo/include/functions_user.inc.php(595): query2array('\nSELECT cat_id\n...', NULL, 'cat_id') #1 /var/www/piwigo/include/functions_user.inc.php(420): calculate_permissions(NULL, NULL) #2 /var/www/piwigo/include/functions_user.inc.php(306): getuserdata(114, true) #3 /var/www/piwigo/include/user.inc.php(75): build_user(114, true) #4 /var/www/piwigo/include/common.inc.php(162): include('/var/www/piwigo...') #5 /var/www/piwigo/index.php(26): include_once('/var/www/piwigo...') #6 {main} thrown in /var/www/piwigo/include/dblayer/functions_mysqli.inc.php on line 906
```

**Comment by Kipjr** [comment]
Solved. During a migration LastModified-values were "0000-00-00 00:00:00", which is now unacceptable (I guess since PHP7?). This was fixed by replacing the values by more recent dates.

**Comment by plegall** [comment]
It is more related to MySQL 5.7. Will be fixed by #670
